freqtrade/exchange/binance.py

```diff
@@ -136,17 +135,19 @@ def fill_leverage_brackets(self):
                 else:
                     leverage_brackets = self._api.load_leverage_brackets()
 
-                for pair, brackets in leverage_brackets.items():
-                    self._leverage_brackets[pair] = [
-                        [
-                            min_amount,
-                            float(margin_req)
-                        ] for [
-                            min_amount,
-                            margin_req
-                        ] in brackets
-                    ]
-
+                for pair, brkts in leverage_brackets.items():
+                    [amt, old_ratio] = [None, None]
+                    brackets = []
+                    for [notional_floor, mm_ratio] in brkts:
```

Comment:
I think the docstring should now explain what/how the brackets are now built (basically - what to expect in field 1, 2, 3, 4 ...